index.bs

```diff
@@ -1545,38 +1565,40 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
-### Attestation Presentation Preference enumeration (enum <dfn enum>AttestationPresentationPreference</dfn>) ### {#attestation}
+### Attestation Conveyance Preference enumeration (enum <dfn enum>AttestationConveyancePreference</dfn>) ### {#attestation-convey}
 
-[=[RPS]=] may use the {{AttestationPresentationPreference}} enum to specify their preference regarding attestation types used
+[=[RPS]=] may use {{AttestationConveyancePreference}} to specify their preference regarding [=attestation conveyance=]
```

Comment:
I'd `<dfn>` "attestation conveyance" here or in the header of this section, since this is where it's defined.

Reply:
ok.